test/stack/11-rego-rules.yaml

```diff
@@ -0,0 +1,228 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hdfs-regorules
+  labels:
+    opa.stackable.tech/bundle: "true"
+data:
+  hdfs.rego: |
+    package hdfs
+
+    import rego.v1
+
+    default allow = false
+
+    # HDFS authorizer
+    allow if {
+        some acl in acls
+        matches_identity(input.callerUgi.shortUserName, acl.identity)
+        matches_resource(input.path, acl.resource)
+        action_sufficient_for_operation(acl.action, input.operationName)
+    }
+
+    # HDFS group mapper
+    # This will return the group data in this form:
+    #    "result": {
+    #        "groups": [
+    #            "admin",
+    #            "developers"
+    #        ]
+    #        ...
+    groups := {group |
+        raw = groups_for_user[input.username][_]
```

Comment:
I haven't tested it but this looks like it will return a result without the "groups" subnesting shown in the comment directly above these lines. Are we sure that the output is the same? (I tried to reproduce my sample data based on the UIF output from another example elsewhere, with named fields).

Reply:
I'm not sure to be honest. I can only say the current PR works flawlessly (both the autorizer and the group mapper)
The response might also depend on whether you call http://opa.default.svc.cluster.local:8081/v1/data/hdfs/groups or http://opa.default.svc.cluster.local:8081/v1/data/hdfs

Reply:
Yes, looking at the code I see the rule output matches what the mapper expects. And should the UIF output change, we can always take of that in the rules (much like we do with trimming the trailing slash).

Reply:
c1aaaf6

Reply:
Thanks!